mandatory/2-function-creation.js

```diff
@@ -15,15 +21,24 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
```

Comment:
Great work ! @ULLIE2298 , alternatively, to write fewer lines of code you could have had had all your conditions in the if statement.